catalog/video/catalog-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/netty/UdpStreamProcessor.java

```diff
@@ -111,10 +113,25 @@
 
     private StreamMonitor streamMonitor;
 
+    private long metacardUpdateInitialDelay = DEFAULT_METACARD_UPDATE_INITIAL_DELAY;
+
     public UdpStreamProcessor(StreamMonitor streamMonitor) {
         this.streamMonitor = streamMonitor;
     }
 
+    /**
+     * @param metacardUpdateInitialDelay must be non-null and >=0 and <={@link #MAX_METACARD_UPDATE_INITIAL_DELAY}
+     */
+    public void setMetacardUpdateInitialDelay(Long metacardUpdateInitialDelay) {
```

Comment:
What happens when the metatype is updated on the admin ui? Does this get called and does delay actually get updated?  I think updateCallback needs to be updated as well.